docs-src/docs/guides/walkthrough/intro.md

```diff
@@ -0,0 +1,24 @@
+# An Iterative Walkthrough
+
+This guide aims to take you through three increasingly-complex deployment examples using Layer0.
+Successive sections build upon the previous ones, and each deployment can be completed either through the Layer0 CLI directly, or through Terraform using our custom [Layer0 Terraform Provider](/reference/terraform-plugin).
```

Comment:
Easy enough to add that link and a bit of clarification here. As for more details, we do cover a little bit in that `/reference/terraform-plugin` link, which includes a couple of links to Terraform docs and how to install Terraform. Think we need more? Do you want to have a separate page that talks about installing terraform?